### PR TITLE
Fix for examples in spec.

### PR DESCRIPTION
**Pull request by sergey-shandar**

### CODE REVIEW

File: lib/util/getErrorsFromModelValidation.ts [hunk 1]
```diff
@@ -22,18 +22,19 @@ export function getErrorsFromModelValidation(
...
-      if (scenarios === undefined) {
-        return undefined
-      }
+      const xMsScenarios = operation["x-ms-examples"]
+      const scenario = operation["example-in-spec"]
+      const scenarios = sm.merge(
+        xMsScenarios !== undefined && xMsScenarios.scenarios !== undefined ?
+          xMsScenarios.scenarios :
+          {},
+        scenario !== undefined ? { "example-in-spec": scenario } : {}
```
Comment (vladbarosan) [inline lib/util/getErrorsFromModelValidation.ts]: >scenario !== undefined ? { "example-in-spec": scenario } [](start = 8, length = 56)

nit: maybe extract as a vriable to be cleat that we are just adding this as a scenario name: scenario

File: lib/validators/cloudError.ts [hunk 2]
```diff
@@ -51,21 +52,21 @@ const cloudError: SchemaObject = {
 }
 
 const cloudErrorWrapperName = `${defaultPrefix}CloudErrorWrapper`
-const cloudErrorSchemaName = `${defaultPrefix}CloudErrorSchema`
+// const cloudErrorSchemaName = `${defaultPrefix}CloudErrorSchema`
```
Comment (vladbarosan) [inline lib/validators/cloudError.ts]: can we delete this

File: lib/validators/cloudError.ts [hunk 3]
```diff
@@ -79,26 +80,29 @@ const cloudErrorWrapper: SchemaObject = {
 }
 
 export interface ResponsesAndDefinitions {
```
Comment (vladbarosan) [inline lib/validators/cloudError.ts]: maybe a comment on exlpaining these

File: lib/validators/cloudError.ts [hunk 4]
```diff
@@ -51,21 +52,21 @@ const cloudError: SchemaObject = {
...
 
 /**
- * Models an ARM cloud error schema.
+ * An ARM cloud error schema.
  */
 const cloudErrorSchema = {
   description: "Error response describing why the operation failed.",
-  title: `#/definitions/${cloudErrorSchemaName}`,
+  // title: `#/definitions/${cloudErrorSchemaName}`,
   schema: {
```
[the remark this answers is not in the document]
Comment (sergey-shandar) [reply lib/validators/cloudError.ts]: ResponseObject can't have title

**Pull request merged**